hack/verify_history.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Verifies git commits starting from predefined prefix.
+# Script is meant to be run in CI. Locally it can produce confusing outputs when local "master" branch is not in sync with remote one
+
+PREFIX="UPSTREAM: <[[:alnum:]]+>: openshift:"
```

Comment:
/lgtm

Do we want `<123>` supported? https://jira.coreos.com/browse/CLOUD-622

Reply:
It's already enforced in openshift/cluster-api. I would be rather for relaxing the constraint to allow both `<123>` and `123`. Though, we can do it later.